build.bzl

```diff
@@ -318,6 +318,10 @@ GENERATED_AUTOGRAD_CPP = [
     "torch/csrc/lazy/generated/RegisterLazy.cpp",
 ] + _GENERATED_AUTOGRAD_CPP_HEADERS + GENERATED_LAZY_H
 
+GENERATED_PYTHON_CPP = [
+    "torch/csrc/functionalization/generated/ViewMetaClassesPythonBinding.cpp"
```

Comment:
tbh I would just stick this in the existing `_GENERATED_AUTOGRAD_PYTHON_CPP` to minimize changes to the build system files - that list already includes non-autograd things, like the `torch.*` python bindings.

(changes to the build system scripts have a habit of being more likely to get reverted :p )